### ISSUE TITLE
Make nested commands work a little more intuitively

### ISSUE THREAD

**Issue by jamonholmgren** [infinitered/gluegun#331]
@ryanlntn had some confusion around nested commands. He didn't know we needed the root command included in the folder in order for it to work.

Let's say you needed `foo stories generate` as a command.

```
commands/
  stories/
    generate.ts
```

This will fail, because it doesn't know what `foo stories` is.

However, if you add the root command:

```
commands/
  stories/
    stories.ts
    generate.ts
```

... it works, because now it'll crawl through `stories.ts` and keep going.

We can document that you need a root command, as above.

We could also assume that a folder is an empty no-op command and just add one if it doesn't exist. That would probably be a more intuitive solution, and not particularly hard to do.

Looking for feedback on this one.

**Comment by skellock** [comment]
I like it. That makes a lot of sense.

**Comment by ryanlntn** [comment]
Yeah I'm all for adding a no-op command if one doesn't exist.

**Comment by skellock** [comment]
Creating a `folder` to get hierarchy is magical.

**Comment by jamonholmgren** [comment]
Sounds good, let's do it.

**Comment by jamonholmgren** [comment]
Closed by #352.
